networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/rpc/beaconchain/BeaconChainMethods.java

```diff
@@ -197,7 +197,8 @@ private static Eth2RpcMethod<GoodbyeMessage, GoodbyeMessage> createGoodBye(
           final PeerLookup peerLookup,
           final RpcEncoding rpcEncoding) {
     final BeaconBlocksByRootMessageHandler beaconBlocksByRootHandler =
-        new BeaconBlocksByRootMessageHandler(spec, metricsSystem, recentChainData);
+        new BeaconBlocksByRootMessageHandler(
+            spec, metricsSystem, recentChainData, MAX_REQUEST_BLOCKS_DENEB);
```

Comment:
It's a Deneb constant but it will be applied much earlier. However smaller is not bigger, so it looks safe. Just thoughts aloud. I think it's ok